tests/unit/src/test/scala/tests/SyntaxErrorLspSuite.scala

```diff
@@ -102,6 +102,9 @@ class SyntaxErrorLspSuite extends BaseLspSuite("syntax-error") {
         """|a/src/main/scala/Main.scala:1:8: error: identifier expected but 'object' found.
            |object object A
            |       ^^^^^^
+           |a/src/main/scala/Main.scala:2:1: error: <error> is already defined as object <error>
```

Comment:
@olafurpg This looks like a regression, any idea where it came from?

Reply:
Reported the issue in https://github.com/scalameta/scalameta/issues/2002 - we can take a look at it later.